.github/workflows/on_push_branch__execute_ci_cd.yml

```diff
@@ -12,7 +12,7 @@ on:
 jobs:
   build_deps:
     runs-on: ubuntu-latest
-    container: bitwalker/alpine-elixir-phoenix:latest
+    container: bitwalker/alpine-elixir-phoenix:1.11
```

Comment:
its fine, just a thought: do we really need an image here? the images contains just a couple of build steps, we could just use an ubuntu image.